pykickstart/commands/ostreesetup.py

```diff
@@ -93,3 +97,61 @@ class RHEL8_OSTreeSetup(F21_OSTreeSetup):
 
 class RHEL9_OSTreeSetup(F21_OSTreeSetup):
     pass
+
+class F38_OSTreeSetup(F21_OSTreeSetup):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.bootc = kwargs.get('bootc', False)
+
+    def __str__(self):
+        retval = super().__str__()
+
+        if self.bootc:
+            retval = retval.strip()
+            retval += " --bootc\n"
+
+        return retval
+
+    def _getParser(self):
+        op = super()._getParser()
+        op.add_argument("--bootc",  action="store_true", default=False,
+                        version=F38, help="""
+                        Enable ostree native containers. Use ``--url`` to specify
+                        remote URL to the container in this format.
+
+                        ```--url=quay.io/examplecorp/example:latest```
+
+                        The ``--ref`` and ``--remote`` is not supported in this mode.""")
+        op.add_argument("--osname", required=False, version=F38, help="""
+                        Argument is not required when --bootc is used.""")
+        op.add_argument("--remote", version=F38, help="""
+                        Cannot be used together with ``--bootc`` parameter.""")
+        op.add_argument("--url", required=True, version=F38, help="""
+                        Supports container from a repository URL.
+                        Use ``--bootc`` to specify container from a repository.""")
+        op.add_argument("--ref", required=False, version=F38, help="""
+                        Cannot be used together with ``--bootc`` parameter.""")
+        op.add_argument("--no-signature-verification", dest="nogpg",
+                        action="store_true", version=F38, help="""
+                        Disable signature verification. An alias to ``--nogpg``.""")
+        return op
+
+    def _check_arguments(self):
+        # Avoid combinations with bootc and other arguments
+        if self.bootc:
+            if self.remote:
+                raise KickstartParseError(_("--bootc and --remote cannot be used together"),
+                                          lineno=self.lineno)
+            if self.ref:
+                raise KickstartParseError(_("--bootc and --ref cannot be used together"),
+                                          lineno=self.lineno)
+        # Require arguments when bootc is not used
+        else:
+            super()._check_arguments()
```

Comment:
This should use `F21_OSTreeSetup` instead of super, for consistency with the rest of pykickstart.